tests/test_file.py

```diff
@@ -151,13 +151,13 @@ def test_token_refresh_store_expires_soon(self):
         access_token = '1/3w'
         token_response = {'access_token': access_token, 'expires_in': 3600}
         http = HttpMockSequence([
-            ({'status': str(http_client.UNAUTHORIZED)},
+            ({'status': str(int(http_client.UNAUTHORIZED))},
```

Comment:
These will pass all of the tests if I just use `int` instead of `str`, but I think that it is supposed to be an actual string, so that is why I'm casting as int, then as str.